cli/README.md

```diff
@@ -381,6 +383,33 @@ Set simulation title with specified font size.
 Done
 ```
 
+### unwatch \<node-id\> \[<node-id> ...\]
+
+Disable the watch status for one or more nodes. See [watch](#watch-node-id-node-id-) for details.
+
+### watch \[\<node-id\>\] \[<node-id> ...\]
+
+Enable additional, detailed log messages on selected node(s) only. This can be useful for interactive debugging or 
+inspection of a node's behavior. 
+
+* To see all nodes currently being watched, use "watch" without parameters. 
+* To see all of the additional log messages, set the overall log-level to "debug" 
+using the "-log debug" command line switch.
+* To see most of the additional log messages, set the overall log-level to "info"
+using the "-log info" command line switch. 
+
+```bash
+> watch 1
+Done
+> watch 3 5 6
+Done
+> watch
+1 3 5 6
+> unwatch 1 3 5
```

Comment:
Might be a good idea to allow `unwatch all` command?

Reply:
Agree, added in the latest commit f1598b6de00e9eae0a5464498d38995f851cb068